lib/browser.ts

```diff
@@ -61,6 +62,11 @@ export class Webdriver {
        opt_message?: string) => wdpromise.Promise<any>;
 }
 
+export class ExtendedWebdriver {
```

Comment:
Nit on naming: We are importing `ExtendedWebDriver` interface and also have an `ExtendedWebdriver` class? By chance could this class be called something else? I almost thought these were the same thing because they differ by `d` vs `D`. Maybe `WebDriverExtended`?